packages/core/src/domain/configuration/intakeSites.ts

```diff
@@ -1,8 +1,16 @@
-export const INTAKE_SITE_STAGING = 'datad0g.com'
-export const INTAKE_SITE_FED_STAGING = 'dd0g-gov.com'
-export const INTAKE_SITE_US1 = 'datadoghq.com'
-export const INTAKE_SITE_EU1 = 'datadoghq.eu'
-export const INTAKE_SITE_US1_FED = 'ddog-gov.com'
+export type Site =
+  | 'datadoghq.com'
+  | 'us3.datadoghq.com'
+  | 'us5.datadoghq.com'
+  | 'datadoghq.eu'
+  | 'ddog-gov.com'
+  | 'ap1.datadoghq.com'
+
+export const INTAKE_SITE_STAGING: Site = 'datad0g.com' as Site
+export const INTAKE_SITE_FED_STAGING: Site = 'dd0g-gov.com' as Site
```

Comment:
I don't want to expose these in the autocompletion, they are not public facing

Reply:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Good catch!